editor/readCoreLibraryVersion.js

```diff
@@ -23,7 +23,15 @@ const readModules = () => {
     )
     .flat()
     .filter((file) => file.endsWith("d.ts"))
-    .filter((file) => !moduleNameBlacklist.includes(file));
+    .filter(
+      (file) =>
+        undefined ===
+        moduleNameBlacklist.find((blackListedName) =>
+          file.endsWith(blackListedName)
+        )
+    );
+
+  console.log(moduleFiles);
```

Comment:
I believe this is not really needed.

Reply:
Actually it can stay. This is only done on npm i, and it can be good information because the logic that finds those module files could have bugs possibly.

Reply:
Nvm, removed it. Stupid console log >:)